### DISCUSSION TITLE
Which flutter version used for developpment

### DISCUSSION THREAD

**Discussion by MobiliteDev** [invertase/dart_custom_lint#287] [Q&A] [answered]
Hello,

I'm trying to contribute to this package, but I can't seem to find a compatible Dart version that satisfies all the dependencies. Currently, I'm using dart 3.5.4 and i tried with dart 3.5.0 but when I run flutter pub get for custom_lint_core, I still encounter the following issue:

```
Because analyzer >=6.9.0 depends on macros >=0.1.3-main.0 <0.1.4 which depends on _macros 0.3.3 from sdk, analyzer >=6.9.0 requires _macros 0.3.3 from sdk.
So, because no versions of _macros from sdk match 0.3.3 and custom_lint_core depends on analyzer ^6.11.0, version solving failed.
```

Could someone please help me figure out the appropriate Flutter version or provide guidance on how to resolve this dependency issue?

Thanks!

**Comment by rrousselGit** [comment, marked as answer]
Use the master channel
